app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -269,33 +269,35 @@ private fun WallpaperThumbnailItem(
 }
 
 @Composable
+@Suppress("MagicNumber")
 private fun WallpaperLogoSwitch(
     checked: Boolean,
     onCheckedChange: (Boolean) -> Unit
 ) {
-    Column(
-        modifier = Modifier.padding(horizontal = 12.dp, vertical = 16.dp),
+    Row(
+        horizontalArrangement = Arrangement.SpaceBetween,
+        modifier = Modifier
+            .padding(horizontal = 12.dp, vertical = 16.dp)
+            .fillMaxWidth()
     ) {
-        Row(
-            horizontalArrangement = Arrangement.SpaceBetween,
-            modifier = Modifier.fillMaxWidth()
-        ) {
-            Text(
-                text = stringResource(R.string.wallpaper_tap_to_change_switch_label_1),
-                color = FirefoxTheme.colors.textPrimary,
-                fontSize = 18.sp,
-                modifier = Modifier.padding(start = 4.dp)
-            )
-            Switch(
-                checked = checked,
-                onCheckedChange = onCheckedChange,
-                colors = SwitchDefaults.colors(
-                    checkedThumbColor = FirefoxTheme.colors.formSelected,
-                    checkedTrackColor = FirefoxTheme.colors.formSurface,
-                    uncheckedTrackColor = FirefoxTheme.colors.formSurface
-                )
+        Text(
+            text = stringResource(R.string.wallpaper_tap_to_change_switch_label_1),
+            color = FirefoxTheme.colors.textPrimary,
+            fontSize = 18.sp,
+            modifier = Modifier
+                .padding(start = 4.dp)
```

Comment:
Should we also add some end padding here to balance it here? We already enforce this in xml. The risk here is that long strings can run to the 'edge' with no spacing between it and the Switch.

Reply:
While I am here, it's interesting that we have padding defined in the parent `Row` and padding here. Without knowing what the designs look like, I almost feel like we can just add this padding to the Row from 12.dp to 16.p